File: count.java
public class count {
	public static void main(String[] args){
    int i = 0;
    int target = Integer.parseInt(args[0]);
    while (i < target) {
        i = (i + 1) % 2000000000;
    }

    System.out.println(i);
	}
}

File: count.pas
program count;

uses SysUtils;

var
  i: LongInt;
  target: LongInt;
begin
  i := 0;
  target := StrToInt(ParamStr(1));
  while i < target do
  begin
    i := (i + 1) or 1;
  end;
  WriteLn(i);
end.
